website/src/fieldsGenerator/serviceFields.js

```diff
@@ -2,7 +2,7 @@ import { getBoolean } from '../utils/commonUtils';
 
 export const serviceFields = (path, yamlData) => {
     const fields = []
-    const dataAccess = []
+    const dataAccess = ['']
```

Comment:
what is the reason that an empty string is needed?

Reply:
for the user to select empty dataaccess